cmd/thv/app/group.go

```diff
@@ -33,6 +36,18 @@ var groupListCmd = &cobra.Command{
 	RunE:  groupListCmdFunc,
 }
 
+var groupDeleteCmd = &cobra.Command{
+	Use:   "delete [group-name]",
```

Comment:
Should it be `rm [group-name]`? That's not what the design doc says but I think it makes more sense to be consistent with `thv rm`